pkg/security/podsecuritypolicy/provider.go

```diff
@@ -103,6 +103,17 @@ func (s *simpleProvider) CreatePodSecurityContext(pod *api.Pod) (*api.PodSecurit
 		sc.SELinuxOptions = seLinux
 	}
 
+	// This is only generated on the pod level.  Containers inherit the pod's profile.  If the
+	// container has a specific profile set then it will be caught in the validation step.
+	seccompProfile, err := s.strategies.SeccompStrategy.Generate(annotations, pod)
+	if err != nil {
+		return nil, nil, err
+	}
+	if annotations == nil {
+		annotations = map[string]string{}
+	}
+	annotations[api.SeccompPodAnnotationKey] = seccompProfile
```

Comment:
Even if no seccomp options are set on the Pod or PodSecurityPolicy, this will add the annotation (with an empty value). Is that the desired behavior?